angr/block.py

```diff
@@ -222,6 +222,38 @@ def instruction_addrs(self):
         return self._instruction_addrs
 
 
+class SootBlock(object):
```

Comment:
`(object)` is unnecessary for Python 3.

Reply:
Fixed Now

Reply:
This class (as in the case of DefUseAnalysis) was implemented by you a long time ago when you started the Java support. Now it is not used anymore by anyone (as far as I can tell) and it has not been tested with the new codebase. Moreover, the other class that deals with loops (`LoopFinder`) already works with soot IR. I think we can remove this class